elastic/kibana
# [UX] Migrate long task metric to UX

## Summary

Related to elastic/uptime#454.

Changes:

- Deleted long task metric route from apm plugin.

- Refactored long task metric chart to dataView and EsSearch.

- Added unit and e2e tests.

### Checklist

Delete any items that are not applicable to this PR.


### For maintainers


### How to test this PR?

- Build `8.4` branch locally and point to one of `oblt` clusters.
- Log in and enable `Inspect ES Queries` in Management -> Stack Management.
- Navigate to `User Experience`.
- Check that metrics are loaded correctly:
![image](https://user-images.githubusercontent.com/95703246/184903833-be209fd4-f42e-42c6-bd80-f91250af3df8.png)
- Click on top right `Inspect` menu and check that `UxLongTaskMetrics (/internal/bsearch)` is in the query list.
